TASK.md

```
@@ -0,0 +1,314 @@
+Implementation Task Spec: Extend docker-wait-for-dependencies to Support Shell Commands + TCP Shorthand
+
+Objective
+•	Add support for array-form command: in Docker Compose where each item can be:
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 2

(MD010, no-hard-tabs)